Core/clim-basic/geometry/region-composition.lisp

```diff
@@ -325,15 +298,10 @@
                  a)))))))
 
 ;;; paths/areas
-(defmethod region-difference ((a standard-line) (b standard-polygon))
-  (multiple-value-bind (x1 y1) (line-start-point* a)
-    (multiple-value-bind (x2 y2) (line-end-point* a)
-      (differenz-line/polygon x1 y1 x2 y2 b))))
-
-(defmethod region-difference ((a standard-line) (b standard-rectangle))
+(defmethod region-difference ((a line) (b polygon))
   (multiple-value-bind (x1 y1) (line-start-point* a)
     (multiple-value-bind (x2 y2) (line-end-point* a)
-      (differenz-line/polygon x1 y1 x2 y2 b))))
+      (difference-segment/polygon x1 y1 x2 y2 b))))
```

Comment:
This would better fit into the "remove duplicated methods" commit.

Reply:
how so? we do not remove here any duplicated method, we simply rename the utility

Reply:
Hm, I think I did something wrong in the review interface.

This was supposed to reference "- remove duplicated methods calling difference-segment/polygon" in the "geometry: move math utilities to a separate file" commit.

Reply:
I'm still unsure what should be done.